spec/features/staff/fund_managers_can_create_a_fund_level_activity_spec.rb

```diff
@@ -39,6 +39,18 @@
       expect(page.find("option[@selected = 'selected']").text).to eq activity_presenter.tied_status
     end
 
+    scenario "the activity has the appropriate funding organisation defaults" do
+      visit organisation_path(organisation)
+      click_on(I18n.t("page_content.organisation.button.create_fund"))
+
+      fill_in_activity_form
+
+      activity = Activity.where(level: :fund).last
```

Comment:
As I think we only want to query for 1 `activity` record we could change this `where` to a `find_by`. It would be more helpful to get a test failure saying `RecordNotFound` than a  more ambigious error for calling `.last` on an empty array, or if `.last` someday returned a different activity record entirely! Something like this may help:

```
fill_in_activity_form(identifier: 'a-fund-has-a-funding-organisation')

activity = Activity.find_by(identifier: 'a-fund-has-a-funding-organisation')
```

I think we could do this on both feature tests here?